contrib/guix/libexec/build.sh

```diff
@@ -282,6 +277,14 @@ mkdir -p "$DISTSRC"
             ;;
     esac
 
+    # Perform basic security checks on INSTALLED executables.
+    # These checks now happen after installation, so RPATHs point to final locations.
```

Comment:
This comment doesn't seem correct to me, since we check that installed binaries have no `RUNPATH` or `RPATH` set. Maybe drop this line?